package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Low level etcd v2 client written in Javascript with failover support",
   "repository": "mafintosh/etcdjs",
   "dependencies": {
-    "roundround": "0.2.0",
-    "request": "2.51.0"
+    "client-request": "~1.0.0",
```

Comment:
To prevent code from changing behavior out from under you on different days that you install this library. Helps with both developer sanity and production use. Library is tested against a specific version of a module and any change to the module can introduce undesirable behavior. Since not every patch or minor version gets tested for behavior changes, it is safest to ship with the specific version we tested.